packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -53,7 +53,8 @@ async function getSummarizerBackCompat(container: IContainer): Promise<ISummariz
 	return response.value as ISummarizer;
 }
 
-async function createSummarizerCore(
+/** @internal */
+export async function createSummarizerCore(
```

Comment:
I think you almost have to export this function

Reply:
I believe most , if not all our test code make use of createSummarizer instead createSummarizerCore.

Reply:
To be honest, not sure what is your suggestion :) Export, but find better name? Not export? I needed it because I want my loader, and I need my loader, because I needed my runtime factory (I believe)